src/deploy/campaign/get-config.ts

```diff
@@ -93,7 +103,9 @@ export const getConfig = async ({
   }
 
   // Domain Token Values
-  const royaltyReceiver = process.env.ENV_LEVEL !== "dev" ? process.env.ROYALTY_RECEIVER! : zeroVaultAddressConf;
+  const royaltyReceiver = process.env.ENV_LEVEL !== EnvironmentLevels.dev
```

Comment:
the `else` above would be the same as this truth value, `process.env.ENV_LEVEL !== EnvironmentLevels.dev`, can we just merge these two blocks of code?